src/functionalTest/java/uk/gov/hmcts/divorce/caseworker/CaseworkerConfirmServiceFT.java

```diff
@@ -54,6 +56,7 @@ public void shouldSetDueDateTo14DaysAfterServiceDateWhenServiceNotProcessedByPro
 
         assertThatJson(response.asString())
             .when(TREATING_NULL_AS_ABSENT)
+            .when(IGNORING_EXTRA_FIELDS)
```

Comment:
Do we want to be ignore extra fields here instead of confirming what they should be in the json file?

Reply:
Yes - done.